lib/pbench/test/unit/server/test_sync.py

```diff
@@ -109,6 +122,21 @@ def test_update_state_update(self, make_logger, more_datasets):
             "INDEX": {"state": "FAILED", "message": "failed"},
         }
 
+    def test_update_state_update_long(self, make_logger, more_datasets):
+        """Test that sync update allows messages over 255 characters."""
```

Comment:
This is a better docstring than the one that `test_error_long()` has (e.g., it makes it clear why the test uses `"tenletters" * 26`).  If you make changes for some other reason (which I don't see why you would...), it would be good to improve `test_error_long()`'s docstring.

Reply:
While you've tempted me, I think I'll take the minor victory today and get this merged.